### ISSUE TITLE
solr image 

### ISSUE THREAD

**Issue by shahmanash** [mskyttner/ala-docker#3]
the solr image when created does not create the biocache core. the other 2 core bie and bie-offline are created and loaded.

**Comment by mskyttner** [comment]
Checking the core status using curl http://localhost:8983/solr/admin/cores?status gives:
```
Cannot create directory: /opt/solr/example/solr/bie/data-offline/index
Cannot create directory: /opt/solr/example/solr/bie/data/index
```
I will add this test to the Makefile in #4

**Comment by mskyttner** [comment]
https://github.com/AtlasOfLivingAustralia/ala-install/blob/master/ansible/roles/solr/tasks/main.yml lists .jar-files that are needed

**Comment by mskyttner** [comment]
Fixed in commit 5a9037f
